lib/pengine/complex.c

```diff
@@ -752,7 +752,7 @@ pe__unpack_resource(xmlNode *xml_obj, pcmk_resource_t **rsc,
     }
 
     value = g_hash_table_lookup((*rsc)->meta, XML_RSC_ATTR_MANAGED);
-    if (value != NULL && !pcmk__str_eq("default", value, pcmk__str_casei)) {
+    if (value != NULL) {
```

Comment:
We can definitely do that. I wouldn't have bothered in this case simply because there's no reason anyone should have known about it or used it. "Never documented" could reasonably mean "no contract."

Reply:
Created T742 if we must keep it :)